app/models/local_authority.rb

```diff
@@ -6,6 +6,7 @@ class LocalAuthority < ApplicationRecord
     has_many :planning_applications
     has_many :constraints
     has_many :contacts
+    has_many :informatives, class_name: "LocalAuthorityInformative"
```

Comment:
If you namespaced the model as `LocalAuthority::Informative` then you wouldn't need this `:class_name` option and the table name is still `local_authoriy_informatives`.